src/entry.ts

```diff
@@ -243,7 +245,7 @@ class Entry {
    * Serialize an entry to a standard format for any transports, e.g. agents.
    * Read more: https://cloud.google.com/logging/docs/structured-logging
    */
-  toStructuredJSON(projectId = '') {
+  toStructuredJSON(projectId = '', useMessageField = true) {
```

Comment:
I'm a bit confused by this. Will end users be setting `useMessageField` manually? I don't think other languages work this way

Reply:
`useMessageField` will be set through options for those users who does not want to have extra "message" field in `jsonPayload` while running in Cloud Run/Functions and using `stdout` logging. Indeed, other languages dont have this bug - I am fixing this just to make sure not to break existing users which use `LogSync` and already get used to the extra "message" field in `jsonPayload`.

Reply:
Yep - for next major release we will turn this flag to be false by default